src/components/FocusTrap/SCREENS_WITH_AUTOFOCUS.ts

```diff
@@ -10,6 +11,14 @@ const SCREENS_WITH_AUTOFOCUS: string[] = [
     SCREENS.SETTINGS.PROFILE.PRONOUNS,
     SCREENS.NEW_TASK.DETAILS,
     SCREENS.MONEY_REQUEST.CREATE,
+    SCREENS.SIGN_IN_ROOT,
 ];
 
-export default SCREENS_WITH_AUTOFOCUS;
+function getScreenWithAutofocus(isAuthenticated: boolean) {
+    if (!isAuthenticated) {
+        return [...SCREENS_WITH_AUTOFOCUS, NAVIGATORS.BOTTOM_TAB_NAVIGATOR];
```

Comment:
sign in page route name is NAVIGATORS.BOTTOM_TAB_NAVIGATOR, but auth screen also has NAVIGATORS.BOTTOM_TAB_NAVIGATOR, so I conditionally added it only if we are not authenticated.